domain/domain.go

```diff
@@ -1121,11 +1123,14 @@ func (do *Domain) updateStatsWorker(ctx sessionctx.Context, owner owner.Manager)
 	dumpFeedbackTicker := time.NewTicker(200 * lease)
 	loadFeedbackTicker := time.NewTicker(5 * lease)
 	statsHandle := do.StatsHandle()
+	idxUsageSyncLease := do.indexUsageSyncLease
+	idxUsageSyncTicker := time.NewTicker(idxUsageSyncLease)
```

Comment:
Why don't we directly use `stats-lease * x` as index usage sync lease? since it is used in `updateStatsWorker`, and this function call has dependency on the fact that `stats-lease` is larger than 0.

Reply:
I think it's better to have independent variables, because strictly speaking, the index usage information is not the statistical information used by planner. I tend to modify it to not be in `updateStatsWorker`. What do you think about this?